### ISSUE TITLE
[BUG] Is there a data source for post race penalties?

### ISSUE THREAD

**Issue by mdsakalu** [theOehrly/Fast-F1#463]
### Describe the issue:

Sprint results from 2023 Qatar are wrong when loading from F1 live timing API due to penalties applied after the finish. Not sure if there are any data sources to allow fastf1 to apply penalties outside of the official FIA documents or ergast.

### Reproduce the code example:

```python
qatar = fastf1.get_event(2023, 17)
sprint = qatar.get_sprint()
sprint.load()
print(sprint.results)
```

**Comment by theOehrly** [comment]
No, there are no other data sources for this data. And given that Ergast does not support the Sprint, it's currently impossible for FastF1 to provide the correct results.

**Comment by theOehrly** [comment]
> @theOehrly Maybe worth adding sprint race support to #445?

It's not mentioned there, but I'm intending to support all session types, yes.
